DSharpPlus.UnifiedCommands/Message/Internals/MessageConvertValues.cs

```diff
@@ -0,0 +1,118 @@
+using DSharpPlus.Entities;
+
+namespace DSharpPlus.UnifiedCommands.Message.Internals;
+
+internal class MessageConvertValues
```

Comment:
argument converters should use an interface and a specialized type for each converter. ideally, we could also hook them into DI to some degree.